docs/management/index-patterns.asciidoc

```diff
@@ -23,79 +18,93 @@ image::images/management-index-read-only-badge.png[Example of Index Pattern Mana
-mapping to list all of the fields that contain a timestamp.
+[float]
+==== Standard index pattern
 
-. Click *Create* to add the index pattern.
+{kib} makes it easy for you to create an index pattern by walking you through 
+the process. Just start typing in the *Index pattern* field and {kib} looks for 
+the names of {es} indices that match your input. If you want to include 
+system indices in your search, toggle the switch in the upper right.
 
-. To designate the new pattern as the default pattern to load when you view the Discover tab, click the *favorite*
-button.
+[role="screenshot"]
+image:management/index-patterns/images/create-index-pattern.png["Create index pattern"]
 
-[float]
-[[set-default-pattern]]
-== Setting the Default Index Pattern
-The default index pattern is loaded automatically when you view the *Discover* tab. Kibana displays a star to the
-left of the name of the default pattern in the Index Patterns list on the *Settings > Indices* tab. The first pattern
-you create is automatically designated as the default pattern.
+You can match multiple indices to create a combination index pattern.  
+Use a comma to separate the names, with no space after the comma. The notation for 
+wildcards (`*`) and the ability to "exclude" (`-`) also apply 
+(for example, `test*,-test3`).
 
-To set a different pattern as the default index pattern:
+When {kib} detects an index with a timestamp, you’re asked to choose a field to 
+filter your data by time. If you don’t specify a field, you won’t be able 
+to use the time filter.
 
-. Go to the *Settings > Indices* tab.
-. Select the pattern you want to set as the default in the Index Patterns list.
-. Click the pattern's *Favorite* button.
+Once you’ve created your index pattern, you can start working with 
+your {es} data in {kib}.  Here are some things to try:
 
-NOTE: You can also manually set the default index pattern in *Advanced > Settings*.
+* Interactively explore your data in Discover.
+* Present your data in charts, tables, gauges, tag clouds, and more in Visualize.
+* Show off your data in a Canvas presentation.
+* If your data includes geo data, visualize it using Maps.
+
+For a walkthrough of creating an index pattern and visualizing the data, 
+see <<getting-started, Getting Started>>. 
 
 [float]
-[[reload-fields]]
-== Reloading the Index Fields List
-When you add an index mapping, Kibana automatically scans the indices that match the pattern to display a list of the
-index fields. You can reload the index fields list to pick up any newly-added fields.
+==== Rolled up index pattern
 
-Reloading the index fields list also resets Kibana's popularity counters for the fields. The popularity counters keep
-track of the fields you've used most often within Kibana and are used to sort fields within lists.
+Clicking *Create index pattern* includes an item for creating a rollup 
```

Comment:
Could we switch these around? For example:

If a rollup index is detected in the cluster, clicking *Create index pattern* includes an item for creating a rollup index pattern.